src/Artsy/Router/RouterLink.tsx

```diff
@@ -25,7 +24,13 @@ export const RouterLink: React.FC<LinkProps> = ({ to, children, ...props }) => {
     return acc
   }, [])
 
-  if (isRouterContext) {
+  const context = useContext(RouterContext)
```

Comment:
This might be overly pedantic, but I think we've established the habit of always putting hooks at the top of the function. There's nothing happening in this code that would break the "hooks must always execute in the same order" rule, but having them at the top of the function makes them easier to find, so that we don't accidentally break that rule in the future.

Reply:
Not at all! Will update.